app/components/Question.tsx

```diff
@@ -14,23 +14,69 @@ export default function Question({
   answer,
 }: QuestionProps) {
   const hasAnswer = answer !== null;
+
+  const handleAnswer = (index: number) => {
+    if (!hasAnswer) {
+      dispatch({ type: "newAnswer", payload: index });
+    }
+  };
+
   return (
-    <div>
-      <h4> {question.question}</h4>
-      <div className=" grid grid-cols-2 gap-4">
-        {question.options.map((option, index) => (
-          <Button
-            variant="contained"
-            className=" cursor-pointer"
-            key={index}
-            disabled={hasAnswer}
-            onClick={() => dispatch({ type: "newAnswer", payload: index })}
-          >
-            {" "}
-            {option}
-          </Button>
-        ))}
-      </div>
-    </div>
+    <Box  sx={{
+      width: 600,
+    }}>
+      <h4 className=" text-2xl font-bold mb-12">
+        {question.question}
+      </h4>
+      <Box display="grid" gridTemplateColumns="1fr" gap={3}>
+        {question.options.map((option, index) => {
+          const isCorrect = index === question.correctOption;
+          const isSelected = index === answer;
```

Comment:
`hasAnswer` and `isSelected` can be merged as one. set isSelected as internal state with it's default value being `hasAnswer`, then on click set the new value as `hasAnswer`